helloworld-html5/README.md

```diff
@@ -63,8 +63,8 @@ The application will be running at the following URL <http://localhost:8080/jbos
 
 You can also test the REST endpoint as follows. Feel free to replace `YOUR_NAME` with a name of your choosing.
 
-* The *XML* content can be tested by accessing the following URL: <http://localhost:8080/jboss-helloworld-html5/hello/xml/YOUR_NAME> 
-* The *JSON* content can be tested by accessing this URL: <http://localhost:8080/jboss-helloworld-html5/hello/json/YOUR_NAME>
+* The *XML* content can be tested by accessing the following URL using HTTP POST: <http://localhost:8080/jboss-helloworld-html5/hello/xml/YOUR_NAME> 
+* The *JSON* content can be tested by accessing this URL using HTTP POST: <http://localhost:8080/jboss-helloworld-html5/hello/json/YOUR_NAME>
```

Comment:
I was reading the README.md and when I hit that lines, I just copy/paste the URLs to the browser. It resulted in error and I thought that something was wrong with the quickstart. But after looking into source code I realized that HTTP POST had to be used. I believe that the current wording is confusing. Since browsers use HTTP GET user must use some other tool or install browser plugin to be able to use that URLs for REST endpoint testing.